routes/teams.js

```diff
@@ -17,10 +18,21 @@ router.get('/', function(res, res) {
 router.get('/team', function(req, res) {
     var term = req.query.q;
     console.log(term);
-    //find the team, This syntax does a sql-type like clause with case insensitivy(sp???) with the RegEx
+    //find the team, This syntax does a sql-type like clause with case insensitivity with the RegEx
     db.collection('teams').find({$or: [{'name': new RegExp(term, 'i')}, {'market': new RegExp(term, 'i')}]}).toArray(function (err, items) {
     	res.json(items);
     });
 });
 
+// grab the players
+router.get('/players', function(req, res) {
+    var term = req.query.q;
+    console.log(term);
+    //find the team, this is getting stuck on uppercase
+    db_players.collection('players').find({$or: [{'full_name': new RegExp(term, 'i')}, {'last_name': new RegExp(term, 'i')}]}).toArray(function (err, items) {
```

Comment:
you know this but this isn't how we will be querying for players. We will be passing a team_id and a limit which will return the players on that team and the limit